metal/i2c.h

```diff
@@ -0,0 +1,112 @@
+/* Copyright 2018 SiFive, Inc */
+/* SPDX-License-Identifier: Apache-2.0 */
+
+#ifndef METAL__I2C_H
+#define METAL__I2C_H
+
+/*! @brief Enums to enable/disable stop condition. */
+typedef enum {
+    METAL_I2C_STOP_DS = 0,
+    METAL_I2C_STOP_EN = 1
```

Comment:
Can you use full DISABLE, ENABLE instead

Reply:
renamed.